### PR TITLE
Added some styling to the scrollbars - uses the theme's predefined variables to match the colors

### PR DESCRIPTION
**Pull request by kaicataldo**
Love the theme (my favorite Atom UI theme yet!), but am not a fan of the default OS scrollbars. Added in some scrollbar styles that match the rest of the theme (color adjusts to theme background color). On a Mac, scrollbars are only visible when you set "Show scroll bars" to "Always" in your "General" settings.

Before:
screen shot 2015-07-26 at 2 40 10 pm

After:
screen shot 2015-07-31 at 12 41 16 am

### CODE REVIEW

File: styles/panels.less [hunk 1]
```diff
@@ -13,6 +13,7 @@ atom-panel {
   }
   &.bottom > *:not(:empty) {
     border-top: 1px solid @tool-panel-border-color;
+    z-index: 3;
```
Comment (kaicataldo) [inline styles/panels.less]: The scrollbar was going over the top border of the bottom pane, causing a weird visual break in the border. This places the bottom pane just over the scrollbar to ensure that doesn't happen.

**Comment by kaicataldo** [comment]
Apologies for closing/reopening the PR

**Comment by kaicataldo** [comment]
Had closed because wasn't sure if you were open to contributions, but opened again because I saw your response to the other PR. Great work on this theme :+1:

**Comment by cwalls251** [comment]
Thanks

**Pull request closed without merging**